lib/tapioca/commands/dsl.rb

```diff
@@ -363,7 +363,8 @@ def underscore(class_name)
 
       sig { params(constant: String).returns(String) }
       def rbi_filename_for(constant)
-        underscore(constant) + ".rbi"
+        # TODO: Figure out a better way than this.
+        underscore(constant).gsub!("rubo_cop", "rubocop") + ".rbi"
```

Comment:
This is definitely not ideal.

Since it's a generated file, why do we care about the name having an underscore or not? Can we let the file be `rubo_cop.rbi`?